scripts/cmux-chat-debug.py

```
@@ -0,0 +1,138 @@
+        if len(buckets["stale"]) > 40:
+            print("  ... +%d more (registry not reconciled)" % (len(buckets["stale"]) - 40))
+    else:
+        print("\n\033[2mstale/foreign: %d (run with --all to list)\033[0m" % len(buckets["stale"]))
+
+
+def main():
+    show_all = "--all" in sys.argv
+    if "--live" in sys.argv:
+        try:
+            while True:
+                os.system("clear")
```

Ruff (0.15.18)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)